parl/layers/layer_wrappers.py

```diff
@@ -13,6 +13,21 @@
 # limitations under the License.
 """
 Wrappers for fluid.layers so that the layers can share parameters conveniently.
```

Comment:
Wrappers for fluid.layers. It helps to easily share parameters between layers.

Reply:
done